WcaOnRails/app/models/attendee_payment_request.rb

```diff
@@ -0,0 +1,6 @@
+class AttendeePaymentRequest < ApplicationRecord
+  belongs_to :receipt, polymorphic: true, optional: true
```

Comment:
You only use `belongs_to` if you yourself are storing the foreign ID. This is not the case here because the model you're introducing does not have a `receipt_id`.

Reply:
Then I will add one ;)